src/domain/human-description/entities/human-description-template.entity.ts

```diff
@@ -93,8 +93,14 @@ export class HumanDescriptionTemplate {
     to: string,
     tokenType: string,
     index: number,
-    args: unknown[],
+    args: readonly unknown[] | undefined,
```

Comment:
I think it'd be better to receive only `readonly unknown[]`.

We can move the check to the caller:

```typescript
if (
  'typeToken' in match.groups &&
  'paramIndex' in match.groups &&
  match.groups.typeToken !== undefined &&
  match.groups.paramIndex !== undefined
) {
  const tokenType = match.groups.typeToken;
  const paramIndex = match.groups.paramIndex;

  if (!args) {
    throw Error(
      `Error mapping token type ${tokenType}. No arguments provided`,
    );
  }

  fragments.push(
    this._mapTokenType(to, tokenType, Number(paramIndex), args),
  );
}
```

Reply:
Good suggestion. I've updated the code accordingly in 694bfe430a6461bfc86766c3e6f7e317abfc656b.